src/+otp/+qg/f.m

```diff
@@ -2,31 +2,31 @@
 
 [nx, ny] = size(L12);
 
-psi = reshape(psi, nx, ny);
+psi = reshape(psi, nx, ny, []);
 
 % Calculate the vorticity
-q = -(Lx*psi + psi*Ly);
+q = -(pagemtimes(Lx, psi) + pagemtimes(psi, Ly));
```

Comment:
Unfortunately, `pagemtimes` is not supported in Octave.  I'm a bit more concerned about the lack or sparse support.  Is this only used to support vectorized psi?

Reply:
Lack of sparse support is due to pagemtimes being built for dlarray. That will go away in the future we assume.

It is used for vectorizing the RHS, but also vectorizing jacobian and adjoint vector products, so it is pretty important. We can add an octave-only stub function to have naive for-loop support, if needed.